numpy/lib/function_base.py

```diff
@@ -157,9 +157,15 @@ def flip(m, axis):
     ----------
     m : array_like
         Input array.
-    axis : integer
-        Axis in array, which entries are reversed.
+    axis : None or int or tuple of ints, optional
```

Comment:
Maybe rewrite this to be `axis : {None, int, Tuple[int]}`?

Reply:
@hameerabbasi: Do we already write it like that elsewhere?

Reply:
@eric-wieser Ah, I thought we did, apparently we don't. It's like this in `np.sum`, for example.